share/templates/07-Band_and_Percussion/01-Concert_Band.mscx

```diff
@@ -100,6 +103,7 @@
         </section>
       <unsorted group="woodwinds"/>
       <section id="brass" brackets="true" showSystemMarkings="false" barLineSpan="true" thinBrackets="true">
+        <family>flugelhorns</family>
```

Comment:
Here is a mismatch between <code>orders.xml</code>. Here are the flugelhorn **above** the cornets while in same order in <code>orders.xml</code> **below** the cornets. Please make the order similar.